### PR TITLE
Consolidate configs from minitest-helper to gem

### PR DESCRIPTION
**Pull request by bousquet**

### CODE REVIEW

File: lib/generators/thincloud/test/templates/factory_girl.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class MiniTest::Spec
+  FactoryGirl.find_definitions
```
Comment (phlipper) [inline lib/generators/thincloud/test/templates/factory_girl.rb]: Move this outside of the class

File: thincloud-test.gemspec [hunk 2]
```diff
@@ -16,16 +16,17 @@ Gem::Specification.new do |s|
...
+  s.add_dependency "rails",                   "~> 3.2.8"
+  s.add_dependency "cane",                    "~> 2.3.0"
+  s.add_dependency "database_cleaner",        "~> 0.8.0"
+  s.add_dependency "guard",                   "~> 1.4.0"
+  s.add_dependency "factory_girl_rails",      "~> 4.1.0"
+  s.add_dependency "minitest",                "~> 3.5.0"
+  s.add_dependency "guard-minitest",          "~> 0.5.0"
+  s.add_dependency "minitest-rails-shoulda",  "~> 0.3.0"
+  s.add_dependency "rb-fsevent",              "~> 0.9.1"
+  s.add_dependency "simplecov",               "~> 0.6.4"
```
Comment (phlipper) [inline thincloud-test.gemspec]: bump me to 0.7.x

File: thincloud-test.gemspec [hunk 3]
```diff
@@ -16,16 +16,17 @@ Gem::Specification.new do |s|
...
+  s.add_dependency "mocha",                   "~> 0.12.5" # Must be after minitest
```
Comment (phlipper) [inline thincloud-test.gemspec]: bump to 0.12.7

File: lib/generators/thincloud/test/templates/minitest_helper.rb [hunk 4]
```diff
@@ -11,23 +11,9 @@
...
-
-# Uncomment if you want Capybara in accceptance/integration tests
-# require "minitest/rails/capybara"
-
-require "mocha"  # mocks and stubs
-
-# use factories instead of fixtures
-require "factory_girl"
-FactoryGirl.find_definitions
+require "thincloud-test"
```
Comment (elskwid) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: We're forcing anyone that uses thincloud-test to use our setup here?

Comment (phlipper) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: It _is_ opinionated software... What is an alternative?

Comment (elskwid) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: More specifically, does the helper template give more flexibility than having these requirements buried in the gem?

Comment (bousquet) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: I see thincloud-test as our configuration preferences for a test harness.

Comment (bousquet) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: You could not require "thincloud-test" in the helper and manually load the components you want.

Comment (elskwid) [inline lib/generators/thincloud/test/templates/minitest_helper.rb:17]: I'm not suggesting we cover every possibility. Maybe I should have said what I like about this …. we don't have to worry about updating the helper when we change thincloud-test.

File: lib/generators/thincloud/test/templates/minitest.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+class MiniTest::Rails::ActiveSupport::TestCase
+
+  # Register any classes that respond to validate to a specific test class
```
Comment (elskwid) [inline lib/generators/thincloud/test/templates/minitest.rb:3]: Hooray! :rainbow:

**Comment by phlipper** [comment]
Merging with @elskwid's blessing :pray: :star2:

**Comment by elskwid** [comment]
:star2: okay :star2:

**Pull request merged**